Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -1585,14 +1585,24 @@ estimateHeaderView push state =
                          <> state.data.destination
     ]
     [ textView $
-        [ text (getString CONFIRM_YOUR_RIDE)
+        [ text$ getString CONFIRM_YOUR_RIDE
         , color Color.black800
         , gravity CENTER_HORIZONTAL
         , height WRAP_CONTENT
         , width MATCH_PARENT
         ] 
         <> FontStyle.h1 TypoGraphy
     , estimatedTimeDistanceView push state
+    , textView $
```

Comment:
Better use `commonTextView`

Reply:
that dont have visibility. otherwise we want to add if else.